spec/javascripts/analytics/static-tracker-spec.js

```diff
@@ -80,6 +80,8 @@ describe("GOVUK.StaticTracker", function() {
           <meta name="govuk:format" content="format">\
           <meta name="govuk:need-ids" content="1,2,3">\
           <meta name="govuk:search-result-count" content="1000">\
+          <meta name="govuk:analytics:publishing-government" content="2005-to-2010-labour-government">\
+          <meta name="govuk:analytics:political-status" content="historic">\
```

Comment:
These values aren't specific to analytics, they are human readable and could be consumed by other things. They don't need the analytics name space.